packages/synapse-react-client/src/components/SynapseTable/SynapseTable.tsx

```diff
@@ -284,8 +283,8 @@ export function SynapseTable(props: SynapseTableProps) {
                         key={cell.id}
                         style={{
                           width: cell.column.getSize(),
+                          textAlign: cell.column.columnDef.meta?.textAlign,
```

Comment:
[Per MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/td#align), the `align` attribute on `td` is deprecated, so update this to use the `textAlign` CSS property